app/src/main/java/hr/foi/air/servicesync/ui/screens/LoginScreen.kt

```diff
@@ -118,6 +123,41 @@ fun LoginScreen(
 
                 Button(
                     onClick =  {
+                        /*
```

Comment:
Consider removing commented code if it isn't necessary.

Reply:
Agreed.